examples/imagenet/README.md

```diff
@@ -0,0 +1,131 @@
+# Integrating Zeus with torchvision and ImageNet
+
+This example will demonstrate how to integrate Zeus with `torchvision` and the ImageNet dataset.
+Also, this example will show how to enable Zeus distributed data parallel training mode with Multi-GPU on a single node. 
+
+[`train.py`](train.py) is adapted from [PyTorch's example training code for ImageNet](https://github.com/pytorch/examples/blob/main/imagenet/main.py).
+You can search for `# ZEUS` in [`train.py`](train.py) for noteworthy places that require modification from conventional training scripts.
+Parts related to data parallel is marked with `# DATA PARALLEL`.
+
```

Comment:
About the `cudnn.benchmark` issue, do you it is necessary to add a NOTE here or a NOTE comment in train.py, like `NOTE: Please set cudnn.benchmark = False, otherwise, it will ruin the profiling in the first few iterations`? Or maybe we can say more generally like `"please disable any other profiling while running zeus"`?

Reply:
I vote for the former. And please add that we're going to fix it soon :) Regarding this, can you create an issue? A good solution would be to add a `skip_steps` parameter together with `warmup_steps` and `measure_steps`, so that the profiler just does nothing for `skip_steps` in the beginning and starts normal profiling after that.

Reply:
Sounds great!